VirtoCommerce.OrderModule.Data/Services/CustomerOrderBuilderImpl.cs

```diff
@@ -31,7 +32,7 @@ public virtual orderModel.CustomerOrder PlaceCustomerOrderFromCart(cartModel.Sho
         #endregion
         protected virtual orderModel.CustomerOrder ConvertCartToOrder(cartModel.ShoppingCart cart)
         {
-            var retVal = AbstractTypeFactory<orderModel.CustomerOrder>.TryCreateInstance();
+            var retVal = AbstractTypeFactory<CustomerOrderWorkflow>.TryCreateInstance();
```

Comment:
don't change this line. Do the type casting later, when you really need to have CustomerOrderWorkflow

Reply:
it's fixed